libs/chiliui/chili/headers/util.lua

```diff
@@ -64,26 +64,10 @@ end
 
 function IsRelativeCoord(code)
 	local num = tonumber(code)
-
-	if (type(code) == "string") then
-		return true
-	elseif (num) and ((1/num) < 0) then
-		return true
-	else
+	if num and ((1/num) < 0) then
 		return false
 	end
-end
-
-function IsRelativeCoordType(code)
```

Comment:
This was unused. Maybe I should have kept it, but didn't want to rewrite it blind to match the above.